react-frontend/src/login/InformativeTextComponent.test.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import InformativeTextComponent from "login/InformativeTextComponent";
+
+it("should equal informative text", () => {
+  const { getByText } = render(
+    <InformativeTextComponent
+      InformativeText="Lets Create the Office Positive"
+      encouragementThought="thought"
+    />
+  );
+  expect(getByText("Lets Create the Office Positive")).toHaveTextContent(
```

Comment:
How did this test case help? Please explain.

CC: @mayuriardad

Reply:
@sahilbhatia i cant understand this comment,we write test case for given input value is render or not